### ISSUE TITLE
No new device found when adding new device

### ISSUE THREAD

**Issue by HasseM** [robertklep/name.klep.sonoff#36]
I try to ad a Sonoff S20 with latest Tasmoto firmware. It's my 2nd Tasmota device that I add (1st is a basic module). When adding a S20 device in the Homey App, when requested, I press reboot in the webinterface of tasmota. As soon as the device comes online, I get the message "No new devices found". I am also using the MQTT broker App on Homey. The broker runs fine and with an MQTT Explorer, I can see the S20 reporting messages to the broker. Offcourse the device is not already added....

It looks like one of the following:
- S20 is not correctly recognized 
- Some configuration file has already a trace of the new device but it is not fully added
- It has problems determining what a new device is (it gives the 'no new device' messages exacty on the moment the tasmotor comes online)
- It has problems when adding a 2nd device..

Restarting homey/broker/tasmota devices and such did not help. I just cannot add the S20, no matter what I try.

Let me know if you need additional information.

**Comment by robertklep** [comment]
Is the device configured as an S20 in the Tasmota configuration? Also, "the lastest firmware" might be an issue, some users are having issues getting Tasmota v7 to work with this app.

**Comment by HasseM** [comment]
Yes, I selected S20 but it is actually called S2x in the tasmota firmware. Maybe that it why it doesnt see it?

**Comment by robertklep** [comment]
Yes, that might be the reason. Try configuring (and adding) it as a Sonoff Basic.

**Comment by HasseM** [comment]
If I configure the device as a basic and add a basic then it is working fine (able to add the device)!

**Comment by robertklep** [comment]
In that case, it's indeed the identifier string that isn't being recognized by the app. It's something that changed in recent firmwares (perhaps v6).

**Comment by HasseM** [comment]
Good, thats probably an easy fix for you. For now I can live with configuring an S20 as a basic, no problem.
